public/static/project_planning.svg

```diff
@@ -1,4 +1,4 @@
-<svg viewBox="0 0 1100 263" width="1100" height="263" preserveAspectRatio="xMidYMid meet" xmlns="http://www.w3.org/2000/svg">
+<svg viewBox="0 0 1800 263" width="1100" height="263" preserveAspectRatio="xMidYMid meet" xmlns="http://www.w3.org/2000/svg">
```

Comment:
What's the thought behind this change?

Reply:
I thought that this looked funky when I was as able to load it as is. Here's a screenshot from the #2660 deploy environment:
![Screen Shot 2022-03-09 at 9 09 13 AM](https://user-images.githubusercontent.com/43454804/157459896-8f44fbb0-d892-463d-95dc-fc47e1ce777d.png)

Reply:
Change looks like this:
![Screen Shot 2022-03-09 at 9 21 36 AM](https://user-images.githubusercontent.com/43454804/157460274-a6fe9571-2964-4df3-ac60-37af8cd12d00.png)

Reply:
The cut-off is intentional, each project is the same steps, so it's to symbolize that it repeats endlessly to the right. The sizing is so the font size is legible (matches the font size of the text body), if it's resized the text would become too small.